clickhouse-admin/types/src/lib.rs

```diff
@@ -965,19 +965,87 @@ pub struct ClickhouseKeeperClusterMembership {
     pub raft_config: BTreeSet<KeeperId>,
 }
 
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    Deserialize,
+    Serialize,
+    JsonSchema,
+)]
+#[serde(rename_all = "snake_case")]
+/// Contains information about distributed ddl queries (ON CLUSTER clause) that were
+/// executed on a cluster.
+pub struct DistributedDdlQueue {
+    /// Query id
+    pub entry: String,
+    /// Version of the entry
+    pub entry_version: u64,
+    /// Host that initiated the DDL operation
+    pub initiator_host: String,
+    /// Port used by the initiator
+    pub initiator_port: u16,
+    /// Cluster name
+    pub cluster: String,
+    /// Query executed
+    pub query: String,
+    /// Settings used in the DDL operation
+    pub settings: BTreeMap<String, String>,
+    /// Query created time
+    pub query_create_time: String,
+    /// Hostname
+    pub host: Ipv6Addr,
+    /// Host Port
+    pub port: u16,
+    /// Status of the query
+    pub status: String,
+    /// Exception code
+    pub exception_code: u64,
+    /// Exception message
+    pub exception_text: String,
+    /// Query finish time
+    pub query_finish_time: String,
```

Comment:
Can we parse these into `chrono::DateTime` or something similar so that we can perform analysis in code?